internal/lightcone/erudition/theseriousnessofbreakfast/theseriousnessofbreakfast.go

```diff
@@ -0,0 +1,66 @@
+package theseriousnessofbreakfast
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Check key.Modifier = "the-seriousness-of-breakfast"
+	Buff  key.Modifier = "get-ready"
```

Comment:
better name (or is this the in-game name?). If someone is looking at logs, might not be clear what `get-ready` is

Reply:
Its in-game name